### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by hugihlynsson**
We've seen a rising number of 404 visits as tracked by this report: https://mixpanel.com/project/1592101/view/639432/app/boards#id=3793389&editor-card-id=%22report-41171888%22
This PR fixes most of those links so our docs users can have a lovely 404-free experience ✨

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**…an/stakeholder-domains**](https://docs-of5chwtb3.avo-dev.app/data-design/avo-tracking-plan/stakeholder-domains) | [**…hes/approval-workflows**](https://docs-of5chwtb3.avo-dev.app/data-design/branches/approval-workflows) |
|:-:|:-:|
|<a href="https://docs-of5chwtb3.avo-dev.app/data-design/avo-tracking-plan/stakeholder-domains">Screenshot of /data-design/avo-tracking-plan/stakeholder-domains</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/9_Yk_6lBM1N6MUSUwSg2h7M_vx73ArDTum4foxIGrp8PVpOM4n-l5a_jbX3Z4yhpuse5uYkvc9d68uKLcfhgNw.png">(view full size)</a> |<a href="https://docs-of5chwtb3.avo-dev.app/data-design/branches/approval-workflows">Screenshot of /data-design/branches/approval-workflows</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/vfU70MiwDK9Lpk2jAw1KpF4gCIHga1mUOI0khWzw3J_pXbS9Yd5UIcoGZHvTJ0TgS6HU6idWxJSM91Ab74Q2xg.png">(view full size)</a> |



| [**/data-design/quick-start**](https://docs-of5chwtb3.avo-dev.app/data-design/quick-start) | [**…sign/start-data-design**](https://docs-of5chwtb3.avo-dev.app/data-design/start-data-design) |
|:-:|:-:|
|<a href="https://docs-of5chwtb3.avo-dev.app/data-design/quick-start">Screenshot of /data-design/quick-start</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/Iv7X9SKh2W3grBrOXC-j9vra1qTXPKwaSMca4C0iFE0qLv65EQrfMCfXkpyN8K5cS_IeuC5WT_R5lKm85HMYHg.png">(view full size)</a> |<a href="https://docs-of5chwtb3.avo-dev.app/data-design/start-data-design">Screenshot of /data-design/start-data-design</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/rmUlGl5m3SWtE3dc24y7Bp7hX2U-bZwpTQDbdtadchFQg26FEdfQIXRzxBguPyufkCYTUW-OT4-xrI25xSf8-g.png">(view full size)</a> |



| [**/**](https://docs-of5chwtb3.avo-dev.app/) | [**…/inspector-issues-view**](https://docs-of5chwtb3.avo-dev.app/inspector/inspector-issues-view) |
|:-:|:-:|
|<a href="https://docs-of5chwtb3.avo-dev.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/ZcU_EU2hy2kZDzWEaUQmydUrUjZWlqr39lAICw6ddtSIkPnVrSwy1NkJg7xYLfZ76Nqfb1VATsp9gAanFMsMkQ.png">(view full size)</a> |<a href="https://docs-of5chwtb3.avo-dev.app/inspector/inspector-issues-view">Screenshot of /inspector/inspector-issues-view</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/uzW4u5rio7IfDN9OuMLF5oB4jQRa1jXIfZd9rv7_-5nlk22MZDEGP-SnjsWhBjjqX0ONX9ObdCQSg09tQCevfA.png">(view full size)</a> |



And 2 other routes:
- [**/inspector/start-using-inspector**](https://docs-of5chwtb3.avo-dev.app/inspector/start-using-inspector)
- [**/publishing/import/get-tracking-plan-into-avo**](https://docs-of5chwtb3.avo-dev.app/publishing/import/get-tracking-plan-into-avo)



Commit f41aabbcd4d91baba475ca7e950dd2005179f777 (https://docs-of5chwtb3.avo-dev.app).

File: pages/data-design/quick-start.mdx [hunk 1]
```diff
@@ -113,14 +109,14 @@ Now we're ready to share our suggestions with our team, and ask them for input.
     autoPlay
     controls
     width="100%"
-    src={'/docs/images/data-design/quickstart/invite-member.mp4'}
+    src="/docs/images/data-design/quickstart/invite-member.mp4"
```
Comment (bragur) [inline pages/data-design/quick-start.mdx:112]: Whuuut. Is this auto-formatting here?

**Review by bragur — APPROVED** [review]
Would be nice to have a verify paths plugin for this

File: pages/data-design/quick-start.mdx (hunk 1 above)
Comment (hugihlynsson) [reply pages/data-design/quick-start.mdx:112]: Not just my hands typing...

**Pull request merged**